src/sentry/quotas/redis.py

```diff
@@ -68,17 +70,26 @@ def get_quotas(self, project):
                 window=oquota[1],
                 reason_code='org_quota',
             ),
-        )
+        ]
+        if key:
```

Comment:
Would it be worth checking for a limit `> 0` here? I'm fairly certain it won't populate an actual key in redis but it just seems like extra work.